packages/ilios-common/addon/components/course/publication-menu.js

```diff
@@ -63,53 +64,67 @@ export default class CoursePublicationMenuComponent extends Component {
     return 'notpublished';
   }
 
-  focusOnFirstItem(menuElement) {
-    menuElement.querySelector('button:first-of-type').focus();
+  focusFirstLink = task(async () => {
+    await timeout(1);
+    document.querySelector('.publication-menu .menu button:first-of-type').focus();
```

Comment:
this works since there's only ever one menu open at the same time. 
however, querying from the doc root is brittle.

i'm suggesting bringing the menu element back as input argument, then making the DOM query from that element instead.

would require identifying the menu element from the event target in the click handler -for example,  if the clicked element is nested within the menu ("menu item"), you'll have to traverse the DOM tree up from it.